### PR TITLE
Day2

### PR DESCRIPTION
**Pull request by zoeames**

### CODE REVIEW

**Comment by zoeames** [comment]
@morecallan

**Comment by zoeames** [comment]
@morecallan didn't finish stuck on for loops

**Comment by morecallan** [comment]
you dumb dumb

**Comment by morecallan** [comment]
oops @zoeames

**Comment by morecallan** [comment]
I feel powerful now.

**Comment by morecallan** [comment]
:1st_place_medal:

**Comment by morecallan** [comment]
WHY DO YOU SUCK AT CODE SO MUCH

**Comment by morecallan** [comment]
YOU LEFT A SEMI COLON OFF ON LINE 6

File: day2.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+console.log("reverseFruit", reverseFruit);
+
+var alphaFruit = fruit.sort();
+console.log("alphaFruit", alphaFruit);
+
+var nums = [9,6,8,2,4,11,66];  //2,4,6,8,9,11,66
+var numSort = nums.sort(
+	function(a,b){
+		return a-b;
+	}
```
Comment (morecallan) [inline day2.js]: indentation is off

Comment (morecallan) [inline day2.js]: fix your indentation

**Comment by zoeames** [comment]
@morecallan i fixed the things

**Comment by morecallan** [comment]
👍

**Comment by morecallan** [comment]
:man_juggling:

**Pull request merged**